recipes/importlib_metadata/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "importlib_metadata" %}
+{% set version = "0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f6fb31513ab8cf7a1fceb5d67041989f05c0c3c3a54998328e257943627121e3
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
```

Comment:
Please update to latest standard for `script`:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33

Reply:
I deliberately dropped --no-cache-dir . More discussion at https://github.com/conda/conda-build/issues/3094

Reply:
ok, thanks for pointing me to that.